catalog/app/containers/Bucket/PackageDialog/PackageDialog.tsx

```diff
@@ -575,16 +576,26 @@ function useFetchEntriesSchema(workflow?: workflows.Workflow) {
 }
 
 export interface ValidationEntry {
+  conflict?: string
   logical_key: string
   size: number
   meta?: JsonRecord
 }
 
+export class EntryNameError extends BaseError {
+  static data: ValidationEntry
```

Comment:
i don't quite get it why this is needed (or at least why it's `static`)

Reply:
This is used for showing in which file there was an error. Under the FilesInput
`static` is a mistyping copypasta, thanks